gdx/src/com/badlogic/gdx/graphics/g3d/particles/ParticleController.java

```diff
@@ -101,6 +111,16 @@ public void setTransform (float x, float y, float z, float qx, float qy, float q
 		this.scale.set(scale, scale, scale);
 	}
 
+	/** Sets the current world transformation. */
+	public void setWorldTransform (Matrix4 worldTransform) {
+		if (worldTransform == null) {
+			this.worldTransform = null;
+		} else {
+			if (this.worldTransform == null) this.worldTransform = new Matrix4();
```

Comment:
Not sure if this is a good way to do it. Perhaps trying to set a matrix which is null should give NullPointerException? However, if the user wants to change/set the matrix for an entire ParticleSystem (which can have multiple controllers) the user would have to loop over them and create new matrices explicitly.